tests/test_processor.py

```
@@ -1552,25 +1553,46 @@ def recognise(self, path, timeout=None):
                 LocalStore(Path(directory) / "gate.db"), RecordingRelay(relay_calls),
                 LateExactRecognizer(), decision_timeout=2.0,
             )
+
+            def process_in_flight_match():
+                try:
+                    results.append(
+                        processor.process(
+                            (self._jpeg(directory, "close-exact.jpg"),)
+                        )
+                    )
+                except BaseException as error:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `BaseException`

(BLE001)